include/solvers/tracer_scratch_data.h

```diff
@@ -260,6 +283,220 @@ class TracerScratchData
       }
   }
 
+
+  /** @brief Reinitializes the content of the scratch for the internal faces. This is only used for the DG assemblers.
+   *
+   * @param[in] cell The cell over which the assembly is being carried.
+   *
+   * @param[in] face_no The face index associated with the cell
+   *
+   * @param[in] sub_face_no The subface index associated with the face
+   *
+   * @param[in] ncell The neighboring cell
+   *
+   * @param[in] n_face_no The face index associated with the neighboring cell
+   *
+   * @param[in] n_sub_face_no The subface index associated with the neighboring
+   * cell
```

Comment:
For readability (and it doesn't match the definition)
```suggestion
   * @param[in] neigh_cell The neighboring cell
   *
   * @param[in] neigh_face_no The face index associated with the neighboring cell
   *
   * @param[in] neigh_sub_face_no The subface index associated with the neighboring
   * cell
```